build.gradle

```diff
@@ -1,83 +1,61 @@
-
-    implementation('org.springframework.boot:spring-boot-starter-web') {
+    implementation('org.springframework.boot:spring-boot-starter-web:2.5.5') {
         exclude(module: 'spring-boot-starter-tomcat')
     }
-    implementation('org.springframework.boot:spring-boot-starter-undertow')
-    implementation('net.logstash.logback:logstash-logback-encoder:5.3')
-    implementation('org.springframework.boot:spring-boot-starter-json')
-    implementation('org.springframework.boot:spring-boot-starter-actuator')
-    implementation('org.springframework.retry:spring-retry')
-    implementation('org.springframework:spring-aspects')
 
+    implementation 'org.springframework.boot:spring-boot-starter-undertow:2.5.5'
+    implementation 'org.springframework.boot:spring-boot-starter-actuator:2.5.5'
+    implementation 'org.springframework.cloud:spring-cloud-aws-messaging:2.2.6.RELEASE'
+
+    implementation 'net.logstash.logback:logstash-logback-encoder:6.6'
 
     // Override the default version of elasticSearch, we're still using V6
     def elasticSearchVersion = '6.8.3'
     implementation("org.elasticsearch:elasticsearch:${elasticSearchVersion}")
     implementation("org.elasticsearch.client:elasticsearch-rest-client:${elasticSearchVersion}")
-    implementation("org.elasticsearch:elasticsearch-x-content:${elasticSearchVersion}")
-    implementation("org.elasticsearch:elasticsearch-cli:${elasticSearchVersion}")
     implementation("org.elasticsearch:elasticsearch-core:${elasticSearchVersion}")
     implementation("org.elasticsearch:elasticsearch-secure-sm:${elasticSearchVersion}")
 
-    implementation('org.apache.camel:camel-spring-boot:2.24.0')
-    implementation('org.apache.camel:camel-jackson:2.24.0')
-    implementation('org.apache.camel:camel-aws:2.24.0')
-    implementation('org.apache.httpcomponents:httpmime:4.5.6')
-    implementation('com.amazonaws:aws-java-sdk:1.11.553')
-    implementation('javax.xml.bind:jaxb-api:2.3.0')
-
     implementation('org.elasticsearch.client:elasticsearch-rest-high-level-client:6.6.2')
     implementation 'com.github.awslabs:aws-request-signing-apache-interceptor:b3772780da'
 
-    compileOnly 'org.projectlombok:lombok'
-    annotationProcessor 'org.projectlombok:lombok'
+    compileOnly 'org.projectlombok:lombok:1.18.22'
 
-    testImplementation('org.springframework.boot:spring-boot-starter-test')
-    testImplementation("org.apache.camel:camel-test-spring:2.24.0")
-    testImplementation('org.assertj:assertj-core')
+    annotationProcessor 'org.projectlombok:lombok:1.18.22'
+    annotationProcessor'org.springframework.boot:spring-boot-configuration-processor:2.5.5'
+
+    testImplementation 'org.springframework.boot:spring-boot-starter-test:2.5.5'
+    testImplementation 'org.awaitility:awaitility:4.1.0'
+    testImplementation 'org.assertj:assertj-core:3.21.0'
+    testImplementation 'junit:junit:4.13.2'
     testImplementation('net.javacrumbs.json-unit:json-unit-assertj:2.27.0')
```

Comment:
I really dislike that we have assertj, theres no reason not to use the inbuilt in my opinion. But probably out of scope of this...

Reply:
One for a future refactor then...